### PR TITLE
Added ability to show all core temperatures in status line at once.

### PR DESCRIPTION
**Pull request by NickOrr0617**
Also added a check on the core temperatures against the "high" temperature
from sensors and prepending an exclamation point if the temperature of the
core exceeds that value.  This gives a visual warning to the user
that the cores may be overheating.

### CODE REVIEW

File: extension.js [hunk 1]
```diff
@@ -188,6 +197,7 @@ const SensorsMenuButton = new Lang.Class({
             // Configure as main sensor and set panel string
             item.setMainSensor();
             this.statusLabel.set_text(item.getPanelString());
+            //this.statusLabel.set_text(allCoreTemps);
```
Comment (HarlemSquirrel) [inline extension.js]: Did you want this line in here?

**Review by HarlemSquirrel — CHANGES_REQUESTED** [review]

File: extension.js (hunk 1 above)
Comment (NickOrr0617) [reply extension.js]: Doesn't need to be, that was my first crack at it, before I decided to make it configurable I just hacked all the cores in for my personal usage.  I can certainly remove it.  What ever is easiest!

Comment (HarlemSquirrel) [reply extension.js]: OK, it looks good! Could you please remove this line then?

**Comment by NickOrr0617** [comment]
Changes made!

**Review by HarlemSquirrel — APPROVED** [review]

**Pull request merged**